src/main/java/com/gamemoonchul/application/MatchUserService.java

```diff
@@ -21,15 +23,22 @@ public class MatchUserService {
 
     public List<MatchUser> saveAll(List<ParticipantRecord> participants, MatchGame matchGame) {
         List<MatchUser> matchUsers = new ArrayList<>();
-        participants.stream().map(
-                participant -> matchUserConverter.toEntities(participant, matchGame)
-        ).forEach(matchUser -> {
-            matchUsers.add(matchUserRepository.save(matchUser));
-        });
+        participants.stream()
+                .map(
+                        participant -> matchUserConverter.toEntities(participant, matchGame)
+                )
+                .forEach(matchUser -> {
+                    matchUsers.add(matchUserRepository.save(matchUser));
+                });
```

Comment:
```suggestion
                .map(participant -> 
                    matchUserConverter.toEntities(participant, matchGame)
                )
                .forEach(matchUser -> {
                    matchUsers.add(matchUserRepository.save(matchUser));
                });
```
이런식으로 람다 인자? 부분은 첫줄에 두고 내용을 다음줄에 하는 형식으로 어떨까요

Reply:
[033696f](https://github.com/gamemuncheol/gamemuncheol-api/pull/77/commits/033696fa00134c585a8e99d444426f7817327b5c)에서 수정되었습니다